### PR TITLE
View models for Bids#index and Bids#my_bids

### PR DESCRIPTION
**Pull request by jessieay**
* oh yes

### CODE REVIEW

File: app/view_models/bid_list_item.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+class BidListItem
```
Comment (harrisj) [inline app/view_models/bid_list_item.rb]: Good call

File: app/view_models/bid_list_item.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  def amount_to_currency_with_asterisk
+    if bid == bid.auction.lowest_bid
+      "#{amount_to_currency} *"
+    else
+      amount_to_currency
+    end
+  end
+
+  def created_at
```
Comment (harrisj) [inline app/view_models/bid_list_item.rb]: Nice!

**Comment by harrisj** [comment]
Looks good, but this needs a rebase now

**Comment by jessieay** [comment]
@harrisj rebased!

**Comment by harrisj** [comment]
Let's merge!

**Comment by adelevie** [comment]
LGTM! 🚢

**Pull request merged**